etc/inc/dyndns.class

```diff
@@ -1288,6 +1299,33 @@
 						$this->_debug($body);
 					}
 					break;
+				case 'dnsmadeeasy':
+					if (preg_match('/success/i', $data)) {
```

Comment:
I agree that switch cases are more readable, however this seems to be the coding style for this file.